lib/xmlSchemaFaker.js

```diff
@@ -46,17 +55,19 @@ function convertSchemaToXML(name, schema, attribute, indentChar, inden) {
     }
     retVal += '>';
     retVal += childNodes;
-    retVal += '\n</' + name + '>';
+    retVal += `\n${cInden}</${tagPrefix+name}>`;
```

Comment:
@erfemega We can stick to one thing here as we're doing the same thing differently here and at kine no. 37. i.e. `${tagPrefix+name}` or `${tagPrefix}${name}`.